durable-task-step/src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -136,15 +135,9 @@ protected DurableTaskStepDescriptor() {
         }
 
         @Override public void stop(Throwable cause) throws Exception {
-            try {
-                FilePath workspace = getWorkspace();
-                if (workspace != null) {
-                    controller.stop(workspace, getContext().get(Launcher.class));
-                }
-            } finally {
-                if (stopAttempt++ == 1) { // second attempt
-                    getContext().onFailure(cause);
-                }
+            FilePath workspace = getWorkspace();
+            if (workspace != null) {
```

Comment:
That the slave is offline.

(This code was already there, of course, just reindented.)